css/main.css

```diff
@@ -37,6 +37,10 @@ body {
   color: rgb(254,246,234);
   color: rgba(254,246,234,.8);
 }
+.footmrgn {
+  margin-right:5px;
+  margin-bottom:5px;
```

Comment:
Best way to do this? Since most of these are attached to .container, I'm a little concerned on how to get this to work properly.